### DISCUSSION TITLE
Sonner vs Toast

### DISCUSSION THREAD

**Discussion by dsagita** [unovue/shadcn-vue#668] [General]
So what is the difference between the sonner and toast component ?

**Comment by sadeghbarati** [comment]
Well both are for showing `Notification/Toast` 

Sonner is more featured and it has a powerful API, 
However, `radix-vue` Toast is the only unstyled and accessible Toast component that you need to add more functionality to it

Sonner like toast but with `radix-vue` Toast component

https://twitter.com/benjamincanac/status/1778028759112630726
